### DISCUSSION TITLE
GasEstimateMessageGas error: estimating gas used: message execution failed: exit 16, reason: failed to init new actor (RetCode=16)

### DISCUSSION THREAD

**Discussion by Flare183** [filecoin-project/lotus#7944] [Storage Provider] [answered]
### Checklist

- [X] This is **not** a security-related bug/issue. If it is, please follow please follow the [security policy](https://github.com/filecoin-project/lotus/security/policy).
- [X] This is **not** a question or a support request. If you have any lotus related questions, please ask in the [lotus forum](https://github.com/filecoin-project/lotus/discussions).
- [X] This is **not** a new feature request. If it is, please file a [feature request](https://github.com/filecoin-project/lotus/issues/new?assignees=&labels=need%2Ftriage%2Ckind%2Ffeature&template=feature_request.yml) instead.
- [X] This is **not** an enhancement request. If it is, please file a [improvement suggestion](https://github.com/filecoin-project/lotus/issues/new?assignees=&labels=need%2Ftriage%2Ckind%2Fenhancement&template=enhancement.yml) instead.
- [X] I **have** searched on the [issue tracker](https://github.com/filecoin-project/lotus/issues) and the [lotus forum](https://github.com/filecoin-project/lotus/discussions), and there is no existing related issue or discussion.
- [X] I am running the [`Latest release`](https://github.com/filecoin-project/lotus/releases), or the most recent RC(release canadiate) for the upcoming release or the dev branch(master), or have an issue updating to any of these.
- [X] I did not make any code changes to lotus.

### Lotus component

- [ ] lotus daemon - chain sync
- [X] lotus miner - mining and block production
- [ ] lotus miner/worker - sealing
- [ ] lotus miner - proving(WindowPoSt)
- [ ] lotus miner/market - storage deal
- [ ] lotus miner/market - retrieval deal
- [ ] lotus miner/market - data transfer
- [ ] lotus client
- [ ] lotus JSON-RPC API
- [ ] lotus message management (mpool)
- [ ] Other

### Lotus Version

```text
Daemon:  1.13.1-dev+mainnet+git.a460ab84f.dirty+api1.4.0
Local: lotus version 1.13.1-dev+mainnet+git.a460ab84f.dirty
```

### Describe the Bug

When running the commands to start a new miner (`lotus-miner init --owner=f1yqj6msbovje4epdvxfccnhgwmo7g66***** --worker=f1ymt3lwdzm3likcsxdhxm3227zz****** --no-local-storage`)

I get the following error:
`2021-10-11T14:51:59.374Z        INFO    main    lotus-miner/init.go:167 Checking full node sync status
Worker: 73462; Base: 1188582; Target: 1188583 (diff: 1)
State: complete; Current Epoch: 1188583; Todo: 0

Done!
2021-10-11T14:51:59.385Z        INFO    main    lotus-miner/init.go:175 Checking if repo exists
2021-10-11T14:51:59.385Z        INFO    main    lotus-miner/init.go:191 Checking full node version
2021-10-11T14:51:59.385Z        INFO    main    lotus-miner/init.go:202 Initializing repo
2021-10-11T14:51:59.386Z        INFO    repo    repo/fsrepo.go:140      Initializing repo at '/root/.lotusminer'
2021-10-11T14:51:59.388Z        INFO    main    lotus-miner/init.go:417 Initializing libp2p identity
2021-10-11T14:51:59.393Z        INFO    badger  v2@v2.2007.2/levels.go:183      All 0 tables opened in 0s

2021-10-11T14:51:59.413Z        INFO    badger  v2@v2.2007.2/levels.go:183      All 0 tables opened in 0s

2021-10-11T14:51:59.434Z        ERROR   main    lotus-miner/init.go:262 Failed to initialize lotus-miner: creating miner failed:
    main.storageMinerInit
        /root/lotus/cmd/lotus-miner/init.go:542
  - pushing createMiner message:
    main.createStorageMiner
        /root/lotus/cmd/lotus-miner/init.go:714
  - GasEstimateMessageGas error: estimating gas used: message execution failed: exit 16, reason: failed to init new actor (RetCode=16)
2021-10-11T14:51:59.434Z        INFO    main    lotus-miner/init.go:267 Cleaning up /root/.lotusminer after attempt...
ERROR: Storage-miner init failed`

I've tried to add funds into the worker account as you can see below, but that has not helped.

`Address                                    Balance                   Nonce  Default  
f1ymt3lwdzm3likcsxdhxm3227zzjtabo22*********  0.00000006 FIL            0               
f1yqj6msbovje4epdvxfccnhgwmo7g662**********  1.989404746742595259 FIL  5      X `

### Logging Information

```text
lotus log:
`evtsm
cliutil
wallet-ledger
splitstore
storagemrkt
ping
bs:peermgr
markets-rtvl
storedask
auth
chainstore
connmgr
dagstore
main
storageadapter
payment-channel-settler
repo
chainxchg
cli
metrics-prometheus
quic-transport
markets-rtvl-reval
blockstore
alerting
mplex
gen
bitswap_network
vm
rand
dt-chanmon
chunk
dht
system
events
dt-impl
ffi-wrapper
retrieval_network
advmgr
sectors
market_adapter
dagstore/upgrader
providerstates
ipns
paych
routedhost
preseal
sectoraccessor
disputer
blankhost
dht.pb
dht/RtRefreshManager
peerstore
p2p-config
storagemarket_network
amt
importmgr
nat
graphsync_allocator
modules
tarutil
tcp-tpt
relay
storageminer
piecestore
cmds/http
build
panic-reporter
net/conngater
sub
peerstore/ds
client
discovery
pubsub
beacon
ulimit
gs-traversal
miner
wallet
blockservice
partialfile
rpcenc
providers
reuseport-transport
cmds
diversityFilter
addrutil
tracing
rpc
backupds
peermgr
bitswap
routing/record
stream-upgrader
filestore
retrieval-discovery
watchdog
test-logger
metrics
p2pnode
fullnode
dt_graphsync
fsutil
bs:sprmgr
fsm
unixfs
badger
messagesigner
fsjournal
autorelay
retrieval-fsm
conngater
engine
storagemarket_impl
retrievaladapter
node
boguskey
genesis
drand
statemgr
incrt
table
chain
swarm2
hello
builder
stores
pathresolv
data-transfer
retrieval
graphsync
messagepool
graphsync_network
gs_request_executor
gs-asyncloader
mocknet
bs:sess
eventlog
types
ffiwrapper
badgerbs
actors
data_transfer_network
lock
fil-consensus
autonat
net/identify
retrievalmarket_impl
gs-unverifiedbs
cborrrpc
statetree
markets
paramfetch
basichost`

lotus-miner lot list:
`root@filecoin:~# lotus-miner log list
ERROR: could not get API info for StorageMiner: repo directory does not exist. Make sure your configuration is correct

root@filecoin:~# `
```

### Repo Steps

Have a blank environment.
1. Install and run the daemon
2. Run the miner initialize command
3. Get error

**Comment by Shekelme** [comment, marked as answer]
You need BLS wallets (f3...) for both `owner` and `worker`.

**Comment by Flare183** [comment]
> You need BLS wallets (f3...) for both `owner` and `worker`.

Oh I have. No matter how much FIL is in either wallet. I still get the same error.

**Comment by ysh0566** [comment]
> > You need BLS wallets (f3...) for both `owner` and `worker`.
> 
> Oh I have. No matter how much FIL is in either wallet. I still get the same error.

You need BLS wallets (f3...) for worker. Your woker address used  f1ymt3lwdzm3likcsxdhxm3227zz******

**Comment by Reiers** [comment]
Hi @Flare183 

Thanks for the report. The issue you are experiencing lies elsewhere.

Moving this issue to Lotus Discussions for help and troubleshooting.

Closing Lotus issue ticket.

**Comment by LexLuthr** [comment]
@Flare183 Was this problem resolved after changing the wallet type?

**Comment by LexLuthr** [comment]
I am archiving this discussion as there has been no new update from the author for more than 7 days.
